server/routes/activities/manage-allocations/handlers/exclusions.ts

```diff
@@ -61,14 +62,23 @@ export default class ExclusionRoutes {
 
     const schedule = await this.activitiesService.getActivitySchedule(activity.scheduleId, user)
     const slots = mapActivityScheduleSlotsToSlots(schedule.slots)
-    const dailySlots = mapSlotsToDailyTimeSlots(slots, schedule.scheduleWeeks)
+
+    const weeklySlots = mapSlotsToCompleteWeeklyTimeSlots(slots, schedule.scheduleWeeks)
+
+    // Add all slots to an array
+    const daySlotsList = Object.values(weeklySlots).flatMap(dailySlots =>
+      dailySlots.flatMap(daySlots => daySlots.slots.map(slot => `${daySlots.day}${slot}`)),
+    )
+    // Find day slots with more than one occurance
+    const disabledSlots = _.uniq(daySlotsList.filter(s => daySlotsList.filter(s2 => s2 === s).length > 1))
```

Comment:
I've moved this logic out of the template and into the handler. I think it makes the logic a little easier to follow and can be tested against in the handler unit tests.